. CPATH is definitely useless at runtime
. LD_LIBRARY_PATH is also useless in robotpkg. And better not set it anyway.

--- cmake/templates/_setup_util.py.in~	2013-06-22 23:04:10.000000000 +0200
+++ cmake/templates/_setup_util.py.in	2013-06-22 23:15:27.000000000 +0200
@@ -51,8 +51,6 @@
 # subfolder of workspace prepended to CMAKE_PREFIX_PATH
 ENV_VAR_SUBFOLDERS = {
     'CMAKE_PREFIX_PATH': '',
-    'CPATH': 'include',
-    'LD_LIBRARY_PATH' if not IS_DARWIN else 'DYLD_LIBRARY_PATH': @CATKIN_LIB_ENVIRONMENT_PATHS@,
     'PATH': '@CATKIN_GLOBAL_BIN_DESTINATION@',
     'PKG_CONFIG_PATH': @CATKIN_PKGCONFIG_ENVIRONMENT_PATHS@,
     'PYTHONPATH': '@PYTHON_INSTALL_DIR@',
--- cmake/templates/setup.sh.in~	2013-06-18 23:56:13.000000000 +0200
+++ cmake/templates/setup.sh.in	2013-06-22 23:19:46.000000000 +0200
@@ -27,12 +27,6 @@
 
 # make sure to export all environment variables
 export CMAKE_PREFIX_PATH
-export CPATH
-if [ $_IS_DARWIN -eq 0 ]; then
-  export LD_LIBRARY_PATH
-else
-  export DYLD_LIBRARY_PATH
-fi
 unset _IS_DARWIN
 export PATH
 export PKG_CONFIG_PATH
